src/components/AppNavigation/EditCalendarModal.vue

```diff
@@ -164,6 +170,7 @@ export default {
 			this.calendarColor = calendar.color
 			this.calendarNameChanged = false
 			this.calendarColorChanged = false
+			this.isTransparent = calendar.transparency === 'transparent'
```

Comment:
```suggestion
			this.isTransparent = calendar.transparency !== 'opaque'
```

Then we assume the calendar to be transparent when calendar.transparency is undefined.

From the code comment below

> If this property is not present on a calendar collection,
> then the default value CALDAV:opaque MUST be assumed.